extensions/test/ext.repo

```diff
@@ -0,0 +1,5 @@
+[ext]
+name=extensions
+baseurl=http://localhost:9091/
+enabled=1
+gpgcheck=0
```

Comment:
Not a blocker, but in theory we can enable this right?

Reply:
yeah, enabled in latest push